pandas/core/frame.py

```diff
@@ -4818,66 +4818,82 @@ def aggregate(self, func, axis=0, *args, **kwargs):
 
     def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
               result_type=None, args=(), **kwds):
-        """Applies function along an axis of the DataFrame.
+        """
+        Apply a function along an axis of the `Series`.
```

Comment:
Why did you change `DataFrame` by `Series`? The file is `frame.py` I think it's apply for `DataFrame`, isn't it?

Reply:
add this PR review suggestions into commit c593a70c5